x-pack/plugins/maps/common/descriptor_types/source_descriptor_types.ts

```diff
@@ -29,6 +29,7 @@ export type EMSTMSSourceDescriptor = AbstractSourceDescriptor & {
   // id: EMS TMS layer id. Used when !isAutoSelect
   isAutoSelect: boolean;
   lightModeDefault: string;
+  locale?: string;
```

Comment:
Is sourceDescriptor.locale used anywhere? Looks like the value lives in layerDescriptor. If that is the case, please remove locale from source descriptor.

Reply:
Good catch. This is cruft from a previous implementation. Will remove.